python/src/viewer.py

```diff
@@ -886,6 +891,8 @@ def PlotDesign(
     >>> subdivisions = [size]*dim
     >>> fig = PlotDesign(sample, bounds, subdivisions)
     """
+    warnings.warn('PlotDesign is deprecated in favor of DrawPairs', DeprecationWarning)
```

Comment:
No, `PlotDesign` has a completely different purpose and cannot be replaced with `DrawPairs`.